actinia_gdi/api/gdi_processing.py

```diff
@@ -19,11 +36,17 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
-#######
 
-"""
+
 Extension of actinia_cores ephemeral_processing_with_export AsyncEphemeralExportResource to include process chain templates
 """
+
+__license__ = "Apache-2.0"
+__author__ = "Anika Bettge, Sören Gebbert"
+__copyright__ = "Copyright 2016-2018, Sören Gebbert and mundialis GmbH & Co. KG"
```

Comment:
I thing the gdi_processing.py is also from 2019
```suggestion
__copyright__ = "Copyright 2016-2019, Sören Gebbert and mundialis GmbH & Co. KG"
```